### PR TITLE
Fix contribution guide

### PR DESCRIPTION
**Pull request by unnonouno**
I updated contribution guide.
- fix description of major/minor/revision
- fix CI environment especially about AppVeryor
- remove pyformat and isort

### CODE REVIEW

File: docs/source/contribution.rst [hunk 1]
```diff
@@ -27,11 +27,9 @@ In particular, we are using the master branch for our development, and releases
 Releases are classified into three groups: major, minor, and revision.
 This classification is based on following criteria:
 
-* A **major** release contains catastrophic changes on the interface that may break existing user codes.
-* A **minor** release contains additions and modifications on the interface.
-  It may break some existing user codes, though they must be fixed by small efforts.
-* A **revision** release contains changes that does not affect the documented interface.
-  It mainly consists of bug fixes, implementation improvements, and test/document/example updates.
+- **Major update** contains disruptive changes that break the backward compatibility.
+- **Minor update** contains addition and extension to the APIs keeping the supported backward compatibility.
```
Comment (delta2323) [inline docs/source/contribution.rst]: addition**s** and extension**s**

File: docs/source/contribution.rst [hunk 2]
```diff
@@ -27,11 +27,9 @@ In particular, we are using the master branch for our development, and releases
...
+- **Revision update** contains improvements on the API implementations without changing any API specifications.
```
Comment (delta2323) [inline docs/source/contribution.rst]: any API specification.

**Comment by delta2323** [comment]
LGTM except comments.

**Comment by unnonouno** [comment]
I fixed typo.

**Comment by delta2323** [comment]
LGTM :+1:

**Pull request merged**